src/App.scss

```diff
@@ -17,13 +17,3 @@ iframe {
   padding: 20px;
   border-left: 1px solid lightgray;
 }
-
-.movies {
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
-  gap: 20px;
-}
-
-.card-image img {
```

Comment:
you deleted style and your images aren't look good
<img width="512" alt="Screenshot 2023-09-01 at 16 28 11" src="https://github.com/mate-academy/react_movies-list-js/assets/53228259/93ec8a0c-da86-4b76-b7d7-d41c9194cb38">